css/login.css

```diff
@@ -0,0 +1,29 @@
+ody {
+    box-sizing: border-box;
+}
+
+#external {
+    background-image:url(../assets/feedback-banner.jpg);
+        
+}
+
+img {
+/*    float:left;*/
+    height: 20rem;
+    width: 20rem;
+}
+
+
+#fb1 {
+    border-top:2rem;
+}
+
+#fb2 {
+    border-top:2rem;
+}
+
+.clearfix:after {
```

Comment:
`:before` / `:after` is CSS 2.1 syntax. We're currently at CSS 3.0 syntax, which recommends usage of a double colon for pseudo-elements, i.e. `::before` / `::after`. "Why is that?", you ask? Well, ask, and ye shall be answered! :P